engine/runtime/src/main/scala/org/enso/interpreter/instrument/job/ProgramExecutionSupport.scala

```diff
@@ -79,13 +85,28 @@ trait ProgramExecutionSupport {
     }
 
     callStack match {
-      case Nil => ()
+      case Nil =>
+        methodCallsCache
+          .getNotExecuted(executionFrame.cache.getCalls)
+          .forEach { expressionId =>
+            cachedMethodCallsCallback.accept(
+              new ExpressionValue(
+                expressionId,
+                null,
```

Comment:
Use named arguments for the nulls

Reply:
Unfortunately, it's a Java class, and it does not support named arguments.